play/src/containers/GameSessionContainer.tsx

```diff
@@ -14,7 +15,8 @@ import StartPhase2 from '../pages/StartPhase2';
 import { JoinGameState, FinalResultsState } from '../lib/PlayModels';
 
 export default function GameSessionContainer() {
-  const [gameSession, setGameSession] = useState(   // eslint-disable-line @typescript-eslint/no-unused-vars
+  const [gameSession, setGameSession] = useState(
+    // eslint-disable-line @typescript-eslint/no-unused-vars
```

Comment:
If this is temporarily and needs to be clean up, one common practice is adding `// TODO: Remove this` or something with more info so you can easily find them and make sure these are resolved before final release

Reply:
Added ``// TODO`` statements to explain this line and the lines below it